index.html

```diff
@@ -39,7 +43,215 @@ <h1>I'm Alishah. Glad to see you!</h1>
+                        <li class="lang">css</li>
+                        <li class="lang JavaScript">javascript</li>
+                    </ul>
+                    <div class="primarytext4">
+                        <button class="project">See project</button>
+                    </div>
+                </div>
+            </div>
+
+        </section>
+        <section class="about-my-slef">
+            <div class="top">
+                <div class="text-container">
+                    <h2 class="first-text">About <br> Myself</h2>
+                    <p class="second-text">Hello I'm a software developer! I can help you <br>
+                        bulid a product, feature or website.<br>
+                        Look through some of my work and <br>
+                        experience! if you like what you see and <br>
+                        have a project you need coded, don't <br>
+                        hesitate to contact me.
+                    </p>
+                </div>
+
+                <div class="social-media1">
+                    <h3 class="sub-social-media ">LET’S CONNECT</h3>
+
+                    <ul class="socail-media-icon1">
+                        <li> <a href="#"><img src="./Image/Vector (1).png" alt=""></a></li>
+                        <li><a href="#"><img src="./Image/Linkedin icon.png" alt=""></a></li>
+                        <li><a href="#"><img src="./Image/Vector (4).png" alt=""></a></li>
+                        <li><a href="#"><img src="./Image/Vector.png" alt=""></a></li>
+                        <li><a href="#"><img src="./Image/Vector (3).png" alt=""></a></li>
+                    </ul>
+                </div>
+                <div>
+                    <button class="normal-button">Get my resume</button>
+                </div>
+            </div>
+            <div class="list-skills">
+                <ul class="ul-item">
+                    <li>
+                        <ul class="language">
+                            <li>Languages</li>
+                            <li><img src="Image/Disabled.png" alt="bisabled sign-ups"></li>
+                        </ul>
+                    </li>
+                    <li>
+                        <ul>
+                            <li class="web-lang">
+                                <img src="Image/Ellipse 1.png" alt="bisabled sign-ups">
+
+                                <span>JavaScript</span>
+                            </li>
+                            <li class="web-lang">
+                                <img src="Image/Ellipse 1 (1).png" alt="bisabled sign-ups">
+                                <span>HTML</span>
+                            </li>
+                            <li class="web-lang">
+                                <img src="Image/Ellipse 1 (2).png" alt="bisabled sign-ups">
+                                <span>CSS</span>
+                            </li>
+
+                        </ul>
+                    </li>
```

Comment:
- [ ] Please make sure to place this `li` tag in a separate `ul` tag and put it inside the `<li>Languages</li>` element as it's [art of the drop-down for the languages.
- [ ] To clarify it should look something like this:
    - ```
        <ul class="language">
          <li>Languages</li>
          <li><img src="Image/Disabled.png" alt="bisabled signups"></li>
          <ul>
            <li class="web-lang">
                <img src="Image/Ellipse 1.png" alt="bisabled sign-ups">

                <span>JavaScript</span>
            </li>
            <li class="web-lang">
                <img src="Image/Ellipse 1 (1).png" alt="bisabled sign-ups">
                <span>HTML</span>
            </li>
            <li class="web-lang">
                <img src="Image/Ellipse 1 (2).png" alt="bisabled sign-ups">
                <span>CSS</span>
            </li>
         </ul>
       </ul>
     ```